spec/lib/ruby_smb/smb1/packet/smb_header_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+RSpec.describe RubySMB::Smb1::Packet::SmbHeader do
+
+  subject(:header) { described_class.new }
+
+  it { is_expected.to respond_to :protocol }
```

Comment:
Make these specs cover the hardcoded int values instead of simply testing that the attr is in place. See [echo_request_spec.rb](https://github.com/rapid7/ruby_smb/blob/master/spec/lib/ruby_smb/smb2/packet/echo_request_spec.rb#L14) for an example.